software/py/vanilla_parser/__init__.py

```diff
@@ -0,0 +1,8 @@
+import sys,os
+sys.path.append(os.path.join(os.path.dirname(__file__)))
```

Comment:
Is there another way to do this? Can you do something like `from . import foo`?

Reply:
Apparently I can. Thanks Dustin, learnt a new thing today :)

Reply:
Glad I could help